Spiky juice -- Teams page touchups

judeallred opened the pull request:
handling the checklist from https://www.notion.so/glitch/87bda09f85184c1d81e1659aa62e4392?v=9f7963becc9341ed9af502682c6e9eef&p=748711c81e8b44ef88c3f38425324808

judeallred commented on src/presenters/entity-page-projects.jsx:
```diff
@@ -21,24 +19,14 @@ const EntityPageProjects = ({projects, pins, isAuthorized, addPin, removePin, pr
   return (
     <React.Fragment>
-      {!!pinnedVisible && (
+      {(!!pinnedVisible && pinnedProjects.length) ? (
```
this should be another && instead of the ternary operator

judeallred commented on src/presenters/includes/team-analytics.jsx:
```diff
@@ -261,7 +245,6 @@ TeamAnalytics.propTypes = {
   currentUserIsOnTeam: PropTypes.bool.isRequired,
   addProject: PropTypes.func.isRequired,
   myProjects: PropTypes.array.isRequired,
-  projectLimitIsReached: PropTypes.bool.isRequired,
```
there are lingering projectLimitIsReached's in add-team-project.jsx to clean out

judeallred commented on src/presenters/includes/team-project-limit-reached-banner.jsx:
```diff
@@ -1,36 +0,0 @@
-  
-  render() {
-    return (
-      <aside className="inline-banners team-project-limit-reached-banner">
```
need to clean out these styles

judeallred commented on src/presenters/includes/team-upgrade-info-banner.jsx:
```diff
@@ -1,50 +0,0 @@
-    };
-    
-    return (
-      <aside className="inline-banners team-upgrade-banner">
```
need to clean out team-upgrade-banner

judeallred commented on src/presenters/pop-overs/add-team-user-pop.jsx:
```diff
@@ -8,6 +8,58 @@ import UserModel from '../../models/user';
+  /*
+  login: "judeallred"
+  name: "Jude Allred"
+  */
```
comments need a grammar/editorial pass

judeallred commented on src/presenters/pop-overs/add-team-user-pop.jsx:
```diff
@@ -8,6 +8,58 @@ import UserModel from '../../models/user';
+  });
+
+  console.log(results);
+  // Sort results from highest to lowest poitn values:
```
typo

judeallred commented on src/presenters/pop-overs/add-team-user-pop.jsx:
```diff
@@ -8,6 +8,58 @@ import UserModel from '../../models/user';
+    result.points = points;
+  });
+
+  console.log(results);
```
accidental log

judeallred commented on src/presenters/pop-overs/add-team-user-pop.jsx:
```diff
@@ -8,6 +8,58 @@ import UserModel from '../../models/user';
 import Loader from '../includes/loader.jsx';
 import UserResultItem, {InviteByEmail, WhitelistEmailDomain} from '../includes/user-result-item.jsx';
 
+const rankSearchResults = (results, query) => {
```
should rename this to convey the fact that it modifies and mutates the input array instead of returning a copy.

judeallred replied:
aah right, we can change this function into a function that takes a single result plus the query, and returns the point value.  Then we can pass _that_ function diretly into the results.sort function.  
so the end result will be something like:
`results.sort((a, b) => rankResult(b, query) - rankResult(a, query);`

judeallred commented on src/presenters/pop-overs/team-analytics-project-pop.jsx:
```diff
@@ -102,13 +102,13 @@ class TeamAnalyticsProjectPop extends React.Component {
-            <button className="button-small button-tertiary" onClick={togglePopover} disabled={disabled}>
-              {currentProjectDomain}
+            <button className="button-small button-tertiary" onClick={togglePopover}>
+              Filter: {currentProjectDomain ? currentProjectDomain : 'All Projects'}
```
this should be `currentProjectDomain || 'All Projects'`

judeallred commented on src/presenters/pop-overs/upgrade-team-pop.jsx:
```diff
@@ -1,76 +0,0 @@
-    };
-    
-    return (
-      <dialog className="pop-over upgrade-team-pop">
```
upgrade-team-pop -- css to clean up

judeallred commented:
@GregWeil ready for review :-)

GregWeil commented:
Code all looks good, and the search ordering works pretty well

![image](https://user-images.githubusercontent.com/8932174/47745826-f1669780-dc5a-11e8-8613-4fd657dd2d54.png)
There's some extra padding at the bottom of the analytics add project prompt

judeallred commented:
fixed the padding and changed the wording so it would fit in one line:
![image](https://user-images.githubusercontent.com/12502380/47807020-6b595800-dd11-11e8-9ec3-18e2d84d6276.png)
